components/feedstock/stores.js

```diff
@@ -126,7 +139,8 @@ const Store = ({ dataset, color }) => {
               mt: 3,
               mb: 2,
               cursor: dataset.public ? 'pointer' : 'not-allowed', // Changes cursor to indicate disabled state
-              //opacity: dataset.public ? 1 : 0.5, // Reduces opacity to indicate disabled state
+              opacity: dataset.public ? 1 : 0.5, // Grey out the button when disabled
+              pointerEvents: dataset.public ? 'auto' : 'none', // Disables pointer events when dataset is not public
```

Comment:
This is currently obscuring the `cursor` setting above, which is a helpful cue that we've intentionally disabled this button!

Maybe we can drop the `pointerEvents` customization and instead ensure that the `suffix` does not rotate by updating the prop to something like
```es6
            suffix={
              dataset.public ? (
                <RotatingArrow
                  sx={{
                    transform: ['translateY(-6%)'],
                  }}
                />
              ) : (
                <Arrow />
              )
            }
```